health-services/plan-service/src/main/java/digit/service/PlanFacilityService.java

```diff
@@ -0,0 +1,35 @@
+package digit.service;
+
+import digit.repository.PlanFacilityRepository;
+import digit.repository.PlanRepository;
+import digit.web.models.*;
+import org.egov.common.utils.ResponseInfoUtil;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class PlanFacilityService {
+
+    private PlanFacilityRepository planFacilityRepository;
+
+    public PlanFacilityService(PlanFacilityRepository planFacilityRepository) {
+        this.planFacilityRepository = planFacilityRepository;
+    }
+
+    /**
+     * This method processes the requests that come for searching plans.
+     * @param body
+     * @return
+     */
+    public PlanFacilityResponse searchPlanFacility(PlanFacilitySearchRequest body) {
```

Comment:
rename body to planFacilitySearchRequest

Reply:
updated